aggregator_test.go

```diff
@@ -459,6 +459,134 @@ func TestPerformMigrationsTooManyArgs(t *testing.T) {
 	os.Args = oldArgs
 }
 
+// TestHandleCommandStartService checks whether "start-service"
+// command successfully finishes
+func TestHandleCommandStartService(t *testing.T) {
+	*main.AutoMigratePtr = true
+
+	os.Clearenv()
+	mustLoadConfiguration("./tests/tests")
+	oldArgs := os.Args
+	os.Args = []string{os.Args[0], "start-service"}
+
+	go func() {
+		main.Main()
+	}()
+
+	errCode := main.StopService()
+	assert.Equal(t, main.ExitStatusOK, errCode)
+
+	os.Args = oldArgs
+
+	*main.AutoMigratePtr = false
+}
+
+// TestHandleCommandHelp checks whether "help", "print-help"
+// and undefined command (defaults to help) successfully finish
+func TestHandleCommandHelp(t *testing.T) {
+	exitCodeHelp := main.HandleCommand("help")
+	assert.Equal(t, main.ExitStatusOK, exitCodeHelp)
+
+	exitCodePrintHelp := main.HandleCommand("print-help")
+	assert.Equal(t, main.ExitStatusOK, exitCodePrintHelp)
+
+	exitCodeBadHelp := main.HandleCommand("bad-command")
+	assert.Equal(t, main.ExitStatusOK, exitCodeBadHelp)
+}
+
+// TestHandleCommandConfig checks whether "print-config" command
+// finishes successfully
+func TestHandleCommandConfig(t *testing.T) {
+	exitCodeConfig := main.HandleCommand("print-config")
+	assert.Equal(t, main.ExitStatusOK, exitCodeConfig)
+}
+
+// TestHandleCommandEnv checks whether "print-env" command
+// finishes successfully
+func TestHandleCommandEnv(t *testing.T) {
+	exitCodeEnv := main.HandleCommand("print-env")
+	assert.Equal(t, main.ExitStatusOK, exitCodeEnv)
+}
+
+// TestHandleCommandVersion checks whether "print-version-info"
+// command finishes successfully
+func TestHandleCommandVersion(t *testing.T) {
+	exitCodeConfig := main.HandleCommand("print-version-info")
+	assert.Equal(t, main.ExitStatusOK, exitCodeConfig)
+}
+
+// TestHandleCommandMigrations checks whether "migrations"
+// command successfully finishes
+func TestHandleCommandMigrations(t *testing.T) {
```

Comment:
`TestHandleCommandMigrations`, `TestHandleCommandMigration` and `TestHandleCommandMigrate` contain the same code just with 1 different parameter. I would group them into a single test case.